README.md

```diff
@@ -16,12 +16,113 @@ make env
 
 ## Development Server
 To run the development server, create a `local_settings.py` file as a sibling of `settings.py`. Add a `SECRET_KEY` value
-with some random stuff (e.g `SECRET_KEY = 'sjfio3u903RaggleFraggle'`).
+with some random stuff (e.g `SECRET_KEY = 'sjfio3u903RaggleFraggle'` and see "Environment Variables" section).
 
 The Django local development can be run via:
 
 ```bash
 make watch
 ```
 
+Another means to run local is the manage.py from within the wellregistry path:
 
+```bash
+python -m manage runserver
+```
+Note that environment variable must be configured to connect to a postgres database.
+
+To run tests locally:
+
+```bash
+python -m manage test
+```
+
+
+## Environment Variables
+
+    This project has a private companion config project where environment variable
```

Comment:
I think this is unnecessary and would mean that we would need to change this README if we update how we are managing config on deployment. I am also going to be changing how we do this locally. The description of the variables can stay in

Reply:
I am not sure what you mean by "unnecessary." Of course the readme should reflect changes in procedure.

Reply:
It is coupling this repo unnecessarily with the config project. In the future we may pass in environment variables different or even deploy differently (maybe not a docker container). But the Docker file is part of this repo. The config is not. It's enough to say that we can specify tier specific and secret parameters by using environment variables. The README should reflect documentation of code in this repo.